web/src/components/Section.js

```diff
@@ -4,9 +4,7 @@ import React, { Fragment } from 'react';
 import Instruction from './Instruction';
```

Comment:
I didn't actually change this file, in the end - these changes are due to `prettier`

I'd initially put the waypoints into the `Section` and `Subsection` components thinking we'd want them to happen automatically, but there was *just enough* variety in when and whether we wanted waypoints that it became unwieldy.  Rather than beat it into submission, I refactored the waypoints into their own component.

Reply:
That seems like a very smart move!